JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/Services/UserManagement.cs

```diff
@@ -0,0 +1,38 @@
+﻿using JhipsterXamarin.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JhipsterXamarin.Services
+{
+    public  class UserManagement
+    {
+        private IList<UserModel> UserModels { get; set; }
+        private IUserService UserService { get; set; }
+
+        private IAuthenticationService AuthenticationService;
+
+        protected  async Task OnInitializedAsync()
+        {
+            UserModels = await UserService.GetAll();
+        }
+
+        private async Task ActiveUser(UserModel user, bool activated)
+        {
+            user.Activated = activated;
+            await UserService.Update(user);
+        }
+
+        private async Task DeleteUser(string login)
+        {
+            foreach (UserModel userModel in UserModels)
+            {
+                if (userModel.Login == login)
```

Comment:
Use a lambda with LINQ ?

Reply:
It's incomplete i have to change a lot here.